internal/cluster/distribution/eks/eksprovider/workflow/bootstrap_activity.go

```diff
@@ -303,3 +314,50 @@ func userNameFromArn(arn string) string {
 
 	return arn[idx+1:]
 }
+
+func mergeAuthConfigMaps(defaultConfigMap, inputConfigMap string) (*v1.ConfigMap, error) {
+	var defaultAWSConfigMap v1.ConfigMap
+	_, _, err := scheme.Codecs.UniversalDeserializer().Decode([]byte(defaultConfigMap), nil, &defaultAWSConfigMap)
+	if err != nil {
+		return nil, errors.WrapIf(err, "failed to decode config map")
```

Comment:
**Optional**: a qualifier here and 6 lines below such as `default` and `input` in the error message **MAY** help differentiate between issues with any of the config maps.

Reply:
Thanks fixed, good idea.